structured/structured.go

```diff
@@ -27,4 +29,28 @@ type Error struct {
 	Action string
 	// LikelyCause is the likely cause for the error e.g. "Software bug."
 	LikelyCause string
+	// Err is the original error string.
+	Err error
+}
+
+// Error implements the error#Error interface.
+func (e *Error) Error() string {
+	if e == nil {
+		return ""
+	}
+	ret := fmt.Sprintf("\tmoreInfo=%s impact=%s action=%s likelyCause=%s",
+		e.MoreInfo, e.Impact, e.Action, e.LikelyCause)
```

Comment:
Is it possible that the original error string is nil, if so then makes sense

Reply:
yes, it's possible if the user returns a structured.Error type without the Err field populated.